### PR TITLE
Rust 2017 retrospective

### PR DESCRIPTION
**Pull request by aturon**

### CODE REVIEW

**Comment by aturon** [comment]
cc @rust-lang/core

**Comment by joshtriplett** [comment]
I'd like to propose including @nikomatsakis, who pioneered the idea of giving a large and complex RFC (namely, non-lexical lifetimes) a repository of its own, to allow for issues and pull requests.

File: _posts/2017-12-21-rust-in-2017.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- **Infrastructure**
+  - [mark-simulacrum](https://github.com/Mark-Simulacrum), for getting the [perf](https://perf.rust-lang.org/) website to a highly useful state, and for overhauling rustbuild to better support contribution.
+  - [aidanhs](https://github.com/aidanhs/), for coordinating maintenance of crater.
+- **Language**
+  - [withoutboats](https://github.com/withoutboats/), for keeping us focused on the programmer experience and for helping the community navigate discussion around very thorny language design issues.
+  - [cramertj](https://github.com/cramertj/), for keeping us focused on *shipping*, and in particular building consensus around some of the topics where that’s been hardest to find: impl Trait, and module system changes.
+- **Libraries**
+  - [brson](https://github.com/brson/), for envisioning and largely overseeing the Libz Blitz initiative.
+  - [kodraus](https://github.com/dtolnay), for gracefully taking over the Libz Blitz and seeing it to a successful conclusion.
```
Comment (arielb1) [inline _posts/2017-12-21-rust-in-2017.md]: this shouuld point to kodraus, not dtolnay

**Comment by aturon** [comment]
@joshtriplett @arielb1 done

**Comment by aturon** [comment]
@steveklabnik Before widely announcing the post, please make sure that the incremental benchmark image shows up correctly.

**Pull request merged**